### ISSUE TITLE
error when trying to start-server: [WinError 2] The system cannot find the file specified

### ISSUE THREAD

**Issue by TBAIKamine** [GrahamDumpleton/mod_wsgi#699]
Hello sir, thank you for what you're doing to the community
successfully installed the module by following the instructions for windows users using pip
I had a small issue, it couldnt find apache location first so I set the environment variable MOD_WSGI_APACHE_ROOTDIR
I have WAMP installed (from wampserver.com) and working fine.
when I try this command to start the wsgi server: mod_wsgi-express start-server
I get this:
```
C:\wamp64\www>mod_wsgi-express start-server
Server URL         : http://localhost:8000/
Server Root        : C:/Users/Amine/AppData/Local/Temp/mod_wsgi-localhost-8000-Amine
Server Conf        : C:/Users/Amine/AppData/Local/Temp/mod_wsgi-localhost-8000-Amine/httpd.conf
Error Log File     : C:/Users/Amine/AppData/Local/Temp/mod_wsgi-localhost-8000-Amine/error_log (warn)
Operating Mode     : daemon
Request Capacity   : 5 (1 process * 5 threads)
Request Timeout    : 60 (seconds)
Startup Timeout    : 15 (seconds)
Queue Backlog      : 100 (connections)
Queue Timeout      : 45 (seconds)
Server Capacity    : 20 (event/worker), 20 (prefork)
Server Backlog     : 500 (connections)
Locale Setting     : en_US.cp1252

WARNING: The ability to use the start-server option on Windows
WARNING: is highly experimental and various things don't quite
WARNING: work properly. If you understand a lot about using
WARNING: Python on Windows and Windows programming in general,
WARNING: and would like to help to get it working properly, then
WARNING: you can ask about Windows support for the start-server
WARNING: option on the mod_wsgi mailing list.

Traceback (most recent call last):
  File "C:\Python39\Scripts\mod_wsgi-express-script.py", line 33, in <module>
    sys.exit(load_entry_point('mod-wsgi==4.8.0', 'console_scripts', 'mod_wsgi-express')())
  File "C:\Python39\lib\site-packages\mod_wsgi\server\__init__.py", line 3825, in main
    cmd_start_server(args)
  File "C:\Python39\lib\site-packages\mod_wsgi\server\__init__.py", line 3658, in cmd_start_server
    subprocess.call([executable]+httpd_arguments)
  File "C:\Python39\lib\subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Python39\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Python39\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified

C:\wamp64\www>
```
thank you

**Comment by GrahamDumpleton** [comment]
Don't use ``mod_wsgi-express start-server``. As it says it is highly experimental and various things don't work, including doing it from Windows command shell by the look of it. Have only tried it from Git shell, which is a UNIX style Bourne shell, but which runs on Windows.

Instead, as you have probably found in various places, on Windows run:

```
mod_wsgi-express module-config
```

and copy what it outputs and add that into your Apache configuration file.

Then configure Apache for your specific WSGI application.

In other words you have to manually configure Apache and you don't want to be using ``mod_wsgi-express start-server``.

**Comment by TBAIKamine** [comment]
which I did, copied and pasted the output in apache httpd.conf and restarted apache and the module is loaded. (am sure of that)
however visiting: localhost:8000 returns site can't be reached.
I have two questions after this happened:
1- where the WSGI server is located now ? in my apache www directory or as written above (C:/Users/Amine/AppData/Local/Temp/mod_wsgi-localhost-8000-Amine)
2- what's the default startup .py file ?

in a shared paid webhost I have, there is a root directory with a file named: passenger_wsgi.py
and it's obligatory to have this:
```
import os
import sys

sys.path.insert(0, os.path.dirname(__file__))

def application(environ, start_response):
    start_response('200 OK', [('Content-Type', 'text/html')])
    message = 'hello world'
    return [message]
```
I just edit this file (change the message) and visit it from a browser: mydomainname.com/the_app_directory
and WSGI does the rest (automatically loads passenger_wsgi.py) 
thank you for clarification

**Comment by GrahamDumpleton** [comment]
And did you do any of the configuration talked about in:

* https://modwsgi.readthedocs.io/en/master/user-guides/quick-configuration-guide.html

Phusion Passenger has nothing to do with mod_wsgi.

**Comment by TBAIKamine** [comment]
I can't thank you enough. it worked
I managed to "mount" it after using the WSGIScriptAlias directive
this wasnt mentioned in the pypip where the module is hosted.
it's not an excuse, I should have looked the readthedocs.io too

**Comment by TBAIKamine** [comment]
I can't thank you enough. it worked
I managed to "mount" it after using the WSGIScriptAlias directive
this wasnt mentioned in the pypip where the module is hosted.
it's not an excuse, I should have looked the readthedocs.io too
